stdlib/forwardable.rb

```diff
@@ -1,16 +1,23 @@
+def forwardable_delegate(hash)
```

Comment:
This code produces a global method that is available on every object. Extracting it to the module and including into `Forwardable/SingleForwardable` pollutes ancestors. How about making an inline JS function that takes a `hash` and a `callback` and invokes it with for every `[accessor, method]` pair?

Reply:
> How about making an inline JS function that takes a hash and a callback and invokes it with for every [accessor, method] pair?

It's a good idea 👍